media-libs/libmad/libmad-0.15.1b.recipe

```diff
@@ -46,11 +47,8 @@ BUILD()
 {
 	touch NEWS AUTHORS ChangeLog
 	sed -i 's/AM_CONFIG_HEADER/AC_CONFIG_HEADERS/' configure.ac
-	libtoolize --force --copy --install
-	aclocal-1.13
```

Comment:
you should also adjust the BUILD_PREREQUIRES when you want to use cmd:aclocal (with autoreconf).